src/views/preview/comment/compose-comment.jsx

```diff
@@ -31,16 +31,21 @@ const CommentErrorMessages = {
     isFlood: "Woah, seems like you're commenting really quickly. Please wait longer between posts.",
```

Comment:
I know this is WIP, so just a note.... These strings will all need to be localized. They should be defined in `src/l10n.json` as they'll be used in more than one view (eventually). I would give them keys like `comment.isBad`, to indicate they're comment strings.